data/pom.yml

```diff
@@ -27,59 +27,59 @@ latestFinal:
 # separating 7.x stream below
 
 latest7Final:
-    version: 7.73.0.Final
-    releaseDateV7: 2022-07-22
+    version: 7.74.0.Final
+    releaseDateV7: 2023-06-29
 
-    droolsZip: https://download.jboss.org/drools/release/7.73.0.Final/drools-distribution-7.73.0.Final.zip
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.73.0.Final/droolsjbpm-integration-distribution-7.73.0.Final.zip
-    businessCentralWildFlyWAR: https://download.jboss.org/drools/release/7.73.0.Final/business-central-7.73.0.Final-wildfly23.war
-    kieExecutionServerZip: https://download.jboss.org/drools/release/7.73.0.Final/kie-server-distribution-7.73.0.Final.zip
-    kieWARS: https://repo1.maven.org/maven2/org/kie/server/kie-server/7.73.0.Final/
+    droolsZip: https://download.jboss.org/drools/release/7.74.0.Final/drools-distribution-7.74.0.Final.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.74.0.Final/droolsjbpm-integration-distribution-7.74.0.Final.zip
+    businessCentralWildFlyWAR: https://download.jboss.org/drools/release/7.74.0.Final/business-central-7.74.0.Final-wildfly23.war
+    kieExecutionServerZip: https://download.jboss.org/drools/release/7.74.0.Final/kie-server-distribution-7.74.0.Final.zip
+    kieWARS: https://repo1.maven.org/maven2/org/kie/server/kie-server/7.74.0.Final/
 
     # as part of the migration to JBAke we have a regex based script to update the version
     # as the below is hardcoded to a fixed version, we just inline the value in the content pages for now, so the regex based script won't change them
     # droolsjbpm-tools iss not released any more, last stable version is 7. 46.0.Final
     # latestDroolsJbpmToolsZip: https://download.jboss.org/drools/release/7. 46.0.Final/droolsjbpm-tools-distribution-7. 46.0.Final.zip
 
-    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.73.0.Final/droolsjbpm-tools-distribution-7.73.0.Final.zip
+    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.74.0.Final/droolsjbpm-tools-distribution-7.74.0.Final.zip
 
-    documentationHtmlSingle: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.drools.org/7.73.0.Final/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.drools.org/7.74.0.Final/kie-api-javadoc/index.html
 
-    droolsWhatsNew: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
-    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391094
+    droolsWhatsNew: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
+    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391774
```

Comment:
What about adding an alert-link in the home page too?
https://github.com/kiegroup/drools-website/blob/main/templates/index.ftl#L17

Something like
```html
            <a class="alert-link"
            href="${pom.latestFinal7.droolsReleaseNotesV7}">Drools ${pom.latestFinal7.version} with bugfixes has been released.</a>
```

Reply:
@danielezonca maybe on the download page? If we put it on main page we will have always two alert-links

Reply:
I have no strong opinion, we can remove it when we want of course.
The download page has already the information as far as I know

Reply:
done with c6c3d90

Reply:
![Screenshot 2023-06-30 at 09 06 20](https://github.com/kiegroup/drools-website/assets/1699252/a0736969-d5e3-4680-bb8a-4352ad5c1adc)